python/src/main/python/drivers/piglit-worker.py

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 The GraphicsFuzz Project Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+import sys
+import subprocess
+import shutil
+from subprocess import CalledProcessError
+import time
+
+import gfuzz_common
+import graphicsfuzz_piglit_converter
+
+HERE = os.path.abspath(__file__)
+
+# Add directory above to Python path for access to dependencies.
+# Prepend it so we override any globally installed dependencies.
+sys.path.insert(0, os.path.dirname(os.path.dirname(HERE)))
+
+# noinspection PyPep8
+from fuzzer_service import FuzzerService
+# noinspection PyPep8
+import fuzzer_service.ttypes as tt
+# noinspection PyPep8
+from thrift.transport import THttpClient, TTransport
+# noinspection PyPep8
+from thrift.Thrift import TApplicationException
+# noinspection PyPep8
+from thrift.protocol import TBinaryProtocol
+
+FRAG_SUFFIX = '.frag'
+JSON_SUFFIX = '.json'
+PNG_SUFFIX = '.png'
+SHADERTEST_SUFFIX = '.shader_test'
+PNG_FILENAME = 'shader_runner_gles3000.png'
+LOGFILE_NAME = 'piglit_log.txt'
+STATUS_FILENAME = 'STATUS'
+
+GLXINFO_CMD = ['glxinfo', '-B']
+SHADERRUNNER_CMD = ['shader_runner_gles3']
+SHADERRUNNER_ARG_PNG = '-png'
+SHADERRUNNER_ARG_AUTO = '-auto'
+SHADERRUNNER_ARG_UNIFORMS = '-ignore-missing-uniforms'
```

Comment:
Nit: I think most Python code heavily leans towards splitting words with `_`. `RETURNCODE` is a bit of an exception :-) This could be applied in a few places. I will try to point to most of them.

Reply:
`nodraw` is an example.

Reply:
OK, actually, that is all!

Reply:
I was inconsistent between splitting words before - I've changed the variables to be split now.